java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/InstanceConfigUtils.java

```diff
@@ -43,9 +43,9 @@ public Properties loadServerConfig() {
 
     public Properties loadServerSecrets(String fileName) {
         Properties secrets = new Properties();
-        val resourceUrl = Resources.getResource(fileName);
         ByteArrayInputStream propertyStream = null;
         try {
+            val resourceUrl = Resources.getResource(fileName);
```

Comment:
this was a bug @rblumer4 found when deploying the connector on the build farm (in development we always had a properties file in the classpath)